src/commands/Fun Commands/EmergencyMeetingCommand.ts

```diff
@@ -0,0 +1,20 @@
+import { MessageAttachment } from 'discord.js';
+import { RunFunction } from '../../interfaces/Command';
+
+export const run: RunFunction = async (client, message, args) => {
+	if (!args.length)
+		return message.channel.send(
+			'I need some text to generate this meme! `emergencymeeting <text>`'
+		);
+	const buffer = await client.vacefron.emergencyMeeting(args.join(' '));
+	const attachment = new MessageAttachment(buffer, 'emergencymeeting.png');
```

Comment:
Is this going to write to a file?

Reply:
It didn't on my local machine

Reply:
Let me look into MessageAttachment.

Reply:
Upon looking into it, the second argument can be undefined / null, can you try without the second param and see if it still functions?

Reply:
Yes I will try that

Reply:
Let me know of the results.

Reply:
It does work without the second arg, do you want me to commit the changes?

Reply:
Yes please.